src/Lykke.Job.TransactionHandler/Projections/OperationHistoryProjection.cs

```diff
@@ -147,7 +146,7 @@ public class OperationHistoryProjection
 
         public async Task Handle(ManualTransactionStateSavedEvent evt)
         {
-            await _log.WriteInfoAsync(nameof(OperationHistoryProjection), nameof(ManualTransactionStateSavedEvent), evt.ToJson(), "");
+            _log.WriteInfo(nameof(ManualTransactionStateSavedEvent), evt.ToJson(), "");
```

Comment:
This is not needed with latest version of Lykke.Cqrs - such logging is done from the library